app/uk/gov/hmrc/helptosave/models/UCResponse.scala

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.helptosave.models
+
+import play.api.libs.json.{Format, Json}
+
+case class UCResponse(ucClaimant: String, withinThreshold: Option[String])
```

Comment:
it is more meaningful to have these parameters as Booleans and write a custom reads and writes instance that reads and writes it to "Y" and "N" - this would provide a bit of validation on the response from DWP as well which is missing at the moment